Added Brazillian Portuguese layouts to the keymaps

LuMarquesIlva opened the pull request:
Changes to be committed:
	new file:   brazilian_portuguese-abnt.vim
	new file:   brazilian_portuguese-compact.vim

 The brazilian_portuguese-abnt.vim file needs another additional
 two keys to be added, which I'm unable to do now, the compact is good

LuMarquesIlva committed:
Changes to be committed:
	new file:   brazilian_portuguese-abnt.vim
	new file:   brazilian_portuguese-compact.vim

 The brazilian_portuguese-abnt.vim file needs another additional
 two keys to be added, which I`m unable to do now, the compact is good

LuMarquesIlva committed:
Changes to be committed:
	modified:   runtime/keymap/brazilian_portuguese-abnt.vim
	modified:   runtime/keymap/brazilian_portuguese-compact.vim

 Fixed some things in `compact`, and added the last two keys to `abnt`

LuMarquesIlva committed:
Merge branch 'vim:master' into master

chrisbra commented on runtime/keymap/brazilian_portuguese-abnt.vim:
```diff
@@ -0,0 +1,70 @@
+{U Ù LATIN CAPITAL LETTER U WITH GRAVE
+] [ ASCII OPENING BRACKETS
+} { ASCII OPENING CURLY BRACKETS
+;	ç LATIN SMALL LETTER C WITH CEDILLA
```
style: tab vs spaces.

chrisbra commented on runtime/keymap/brazilian_portuguese-abnt.vim:
```diff
@@ -0,0 +1,70 @@
+} { ASCII OPENING CURLY BRACKETS
+;	ç LATIN SMALL LETTER C WITH CEDILLA
+: Ç LATIN CAPITAL LETTER C WITH CEDILLA
+'	~ ASCII TILDE
```
style: tab vs spaces.

chrisbra commented on runtime/keymap/brazilian_portuguese-compact.vim:
```diff
@@ -0,0 +1,71 @@
+{U Ù LATIN CAPITAL LETTER U WITH GRAVE
+] [ ASCII OPENING BRACKETS
+} { ASCII OPENING CURLY BRACKETS
+;	ç LATIN SMALL LETTER C WITH CEDILLA
```
same here

chrisbra commented on runtime/keymap/brazilian_portuguese-compact.vim:
```diff
@@ -0,0 +1,71 @@
+} { ASCII OPENING CURLY BRACKETS
+;	ç LATIN SMALL LETTER C WITH CEDILLA
+: Ç LATIN CAPITAL LETTER C WITH CEDILLA
+'	~ ASCII TILDE
```
and here

LuMarquesIlva committed:
Changes to be committed:
	modified:   runtime/keymap/brazilian_portuguese-abnt.vim
	modified:   runtime/keymap/brazilian_portuguese-compact.vim
